polkadot/node/network/bridge/src/rx/mod.rs

```diff
@@ -1135,13 +1135,25 @@ async fn dispatch_validation_events_to_all<I>(
 	I: IntoIterator<Item = NetworkBridgeEvent<net_protocol::VersionedValidationProtocol>>,
 	I::IntoIter: Send,
 {
+	macro_rules! send_message {
+		($event:expr, $message:ident) => {
+			if let Ok(event) = $event.focus() {
+				let has_high_priority = matches!(event, NetworkBridgeEvent::PeerViewChange(..));
```

Comment:
Thought about this a bit more, on side effect of this is that we change the order of messages, for example if the system is loaded I can imagine a situation where the bridge sends the `PeerConnected` on the normal channel and then the PeerViewChange on the priority channel, so the PeerViewChange will arrive before the PeerConnected and it will be discarded. 

I don't think it is a frequent problem, but nonetheless something this might cause, so I wonder if we need to make `PeerConnected` high priority as well.

Reply:
Was it possible to receive PeerViewChange before PeerConnected because of some network problems?